submissions/ik-web/js-dom/js/app.js

```diff
@@ -36,44 +36,53 @@ document.addEventListener("DOMContentLoaded", function(e) {
 
     const mainMenu = document.querySelector('.main-menu'); 
     const cardTitle = document.querySelector('.card-title');
+    const burger = document.querySelector('.burger');
     const cardImg = document.querySelector('.card-img');
     const cardText = document.querySelector('.card-text');
 
     //Menu burger
-    document.querySelector('.burger').addEventListener('click', function(e) {
+    burger.addEventListener('click', function(e) {
         e.preventDefault();
         this.classList.toggle('active');
-        document.querySelector('.main-menu').classList.toggle('active');
+        mainMenu.classList.toggle('active');
     });
 
-    //Create a button in main menu
-    let addButtonToMenu = function(item) {
+    //Create a button for main menu
+    const addButtonToMenu = function(item) {
         const li = document.createElement('li');
         const button = document.createElement('button');
+        const buttonName = cars[item].name;
```

Comment:
```suggestion
    const addButtonToMenu = function(buttonName) { // pass buttonName
        const li = document.createElement('li');
        const button = document.createElement('button');
```
What if you will need render another list of buttons, but names will be not from `cars` array ?

Reply:
I've not yet mastered the abstractness of functions very well, but I am beginning to understand them. Thank you.